src/System.CommandLine.Tests/Binding/TypeConversionTests.cs

```diff
@@ -840,13 +892,13 @@ public void Values_can_be_correctly_converted_to_array_of_int_without_the_parser
         [Fact]
         public void Values_can_be_correctly_converted_to_List_of_int_without_the_parser_specifying_a_custom_converter()
         {
-            var option = new Option(
-                "-x",
-                "",
-                new Argument
+            var option = new Option("-x", "")
```

Comment:
Empty description can be removed. Similar comment for the rest of the tests in this file.